### DISCUSSION TITLE
Font Weights not assigned

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1230] [Q&A]
I have noticed that the  font weights , in the styling tab, are very sporadic.

Built in fonts it seems that only 2 different weights are available: Normal and Bold
Google fonts vary widely on the font chosen as to what font weight button changes the weight
Is there someplace that I can set the weights to be applied by those button.... Or can it be set that if a particular weight is NOT available can be disabled and greyed out.  

Also I see we still cannot disable the built in fonts.  I don't want my editors having to search through a long list of fonts and inevitably choosing the wrong font.

**Comment by meetmandeep** [comment]
> Google fonts vary widely on the font chosen as to what font weight button changes the weight

This sums it up. If your Google or Custom Font supports different weights then the Styling Tab will reflect that. 

> Is there someplace that I can set the weights to be applied by those button.... Or can it be set that if a particular weight is NOT available > can be disabled and greyed out.

You have full control over the generated stylesheet. You can write your own CSS or Sass. See: https://www.youtube.com/watch?v=TwvySHpffyM

> Also I see we still cannot disable the built in fonts.
You can now manage fonts from Design > Themes > Settings > Custom > Settings (Gear Icon) > Fonts

You may add Google or Custom Fonts or remove existing fonts.

**Reply by scottw12** [reply]
How do I remove all but the last three fonts. I do not see the system fonts at Design > Themes > Settings > Custom > Settings (Gear Icon) > Fonts 

I understand that not all fonts ,especially google fonts, do not have all weights represented. However the main Google font I am using is Roboto and it indeed has representation of all the listed weights.  https://fonts.google.com/specimen/Roboto?query=roboto

Update:  I looked in Chrome debugger and indeed the font weights changed 100 300 400 500 700 and 900. 

But the font displayed in the content screen only displays three different weights as you change the buttons, not all six. making it difficult to use. Its late now but in the morning I will see if the font actually gets saved when published as set. Still it makes layout  awkward if you do not see the font changes at the time you set them.
